<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-3UIEUVU9" failures="1" tests="1" name="Default test" time="39.962" errors="0" timestamp="2022-03-23T18:55:12 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="39.105">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Lead  found in lead list">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Lead  found in lead list
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at zoho.managers.WebDriverManager.logFailure(WebDriverManager.java:66)
at pages.LeadDetailPage.validateLeadNotPresent(LeadDetailPage.java:30)
at zoho.teststeps.Leads.verifyLeadCreation(Leads.java:103)
at âœ½.Lead 'John Alex' should 'not be present' inside the grid(file:///D:/LearningJavaSelenium/CucumberLearning/src/test/resources/zoho/leads.feature:50)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
